html/css/widgets-styles.css

```diff
@@ -0,0 +1,23 @@
+.foreign {
```

Comment:
Can you use a more specific class name that foreign? Maybe content-editable-foreign or something?

Maybe in the future we need foreign objects for other use cases, and in any case in the present it was hard for me to figure out if they had something to do with content editables or not